test/e2e/e2e_test.go

```diff
@@ -767,6 +857,39 @@ func TestE2E(t *testing.T) {
 		}
 	})
 
+	// make sure that deleting service-cert-secret regenerates a working secret again
+	t.Run("serving-cert-secret-delete-data", func(t *testing.T) {
+		serviceName := "metrics"
+		namespace := "openshift-service-ca-operator"
+		ns, err := createTestNamespace(adminClient, "test-"+randSeq(5))
```

Comment:
Given that only the name of the created namespace is used, maybe the following?

```golang
testNamespaceObj, err := ...
testNamespace := testNamespaceObj.Name
```

Reply:
Done!